docs/generated/galasactl_auth_tokens_get.md

```diff
@@ -13,7 +13,8 @@ galasactl auth tokens get [flags]
 ### Options
 
 ```
-  -h, --help   Displays the options for the 'auth tokens get' command.
+  -h, --help          Displays the options for the 'auth tokens get' command.
+      --user string   An optional flag that is used to retrieve the access tokens of the currently logged in user. The input must be a string.
```

Comment:
Looking better but still mentions the `currently logged in user`. Maybe something like:
```
Optional. Retrieves a list of access tokens for the user with the given username.`
```

(feel free to replace `username` with `login ID` if that makes more sense!)